wcfapp/WcfClient/IService1.cs

```diff
@@ -0,0 +1,12 @@
+﻿using System.ServiceModel;
+
+namespace WcfClient
+{
+    // NOTE: You can use the "Rename" command on the "Refactor" menu to change the interface name "IService1" in both code and config file together.
+    [ServiceContract]
```

Comment:
Cleanup the template comment.

Reply:
I cleaned this up. However, I was intentionally leaving them there. I wanted to show users this is really just from VS template and I am not doing anything fancy.